pkg/popeye.go

```diff
@@ -118,7 +118,8 @@ func (p *Popeye) dump(printHeader bool) error {
 		}
 		fmt.Fprintf(p.outputTarget, "%v\n", res)
 	case report.PrometheusFormat:
-		pusher := p.builder.ToPrometheus(p.flags.PushGatewayAddress)
+		pusher := p.builder.ToPrometheus(p.flags.PushGatewayAddress,
+			p.client.ActiveCluster(), p.client.ActiveNamespace())
 		err := pusher.Add()
```

Comment:
@eminugurkenar Perhaps we can roll a single liner here ie 

```go
if err := pusher.Add(); err != nil {
log.Fatal(...)
}
```

Reply:
as described above it just returns the error now.